docs/cookbook/deployment/introduction-to-sylius-cloud.rst

```diff
@@ -0,0 +1,100 @@
+- \- Multi-services and multi-apps - it doesn't matter if your application is a monolith or microservice-based architecture. It doesn't matter how many different external services you need (MySQL, PostgreSQL, ElasticSearch, Redis, etc.) - it just works well in all cases.
+
+Sylius Cloud Dictionary
+-----------------------
+
+When diving into the Sylius Cloud, it's important to get familiar with some key terms. These terms help you navigate and understand how Sylius Cloud works.
+
+- \- **Sylius Cloud Console** - a web-based interface that allows users to manage their projects, environments, applications, and services from a centralized dashboard. It provides a user-friendly and intuitive interface for performing various tasks related to application development, deployment, and management on the Sylius Cloud platform.
+- \- **Sylius Cloud CLI** - the Command Line Interface (CLI) tool that allows users to interact with their Sylius Cloud projects, environments, applications, and services from the command line. It provides a set of commands and utilities that streamline common tasks related to application development, deployment, and management on the Sylius Cloud platform.
+- \- **Project** - a container for all your environments, applications, and services. When you create a new project, you're essentially setting up a workspace where you can manage your development, staging, and production environments, along with the associated code repositories and services.
+- \- **Environment** - a self-contained instance of your application that represents a specific stage in the software development lifecycle. Environments are used to separate different stages of development, testing, and production, allowing you to manage your application's codebase, configuration, and services in a controlled and isolated manner.
+- \- **Build process** - the series of steps that are executed to prepare your application for deployment to a specific environment. These steps involve installing dependencies (Composer), warming-up application cache, building NodeJS assets, and performing any other tasks necessary to ensure that your application is ready to run in the target environment.
+- \- **Deployment Process** - the process of replacing the existing application with the already built, updated version. The current application version is hosted until the whole deployment process exits without an error.
+- \- **Relations** - the connections established between different services or components within your project. These connections enable communication and interaction between services, allowing them to work together seamlessly to support your application.
+
+How to deploy Sylius to Sylius Cloud by Platform.sh?
+====================================================
+
+.. tip::
+
+    Start with reading `Platform.sh documentation <https://docs.platform.sh/frameworks/symfony.html>`_.
```

Comment:
I would also like to point out that in the `Prepare a Platform.sh project` section, after creating the project, there is an instruction of starting and deploy, where you can go step by step with the correct IDs, etc., of your newly created project. You might consider including a screenshot or something similar to illustrate this. Check it out:

![Screenshot 2024-05-21 at 10 29 18](https://github.com/Sylius/Sylius/assets/53942444/aad0eb51-a96a-4241-833f-b09f660ec0fb)

Reply:
it is a page of the `test` organisation with the `custome project 1` project and standard-with-113 environment that is a main production branch from the https://github.com/Sylius/Sylius/pull/16279/files#diff-7a7f6077d161339cde0de18ff4a9460e131db5bcb33968927327e194fd082248 step